src/cephadm/cephadm

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/python3
 
+VERSION="0.1"
```

Comment:
:-1: for inventing a new version here. please use

https://github.com/ceph/ceph/blob/493f216c1efc6feb459981bf75ca939572099ef3/src/ceph.in#L37-L41

instead. Let's keep cephadm in the same version as ceph itself.

Reply:
@sebastian-philipp : I think that to use the same version as Ceph can make difficult to solve some kind of problems. 

This implies that any modification in cephadm script would imply to change ceph version. 
Seems a little bit radical... and probably not acceptable at all.

and, If we do not change the ceph version:
- how can we check if we are using the new or the old script?
- how can the users know it they are using the right version?

Cephadm script is a file that can be distributed alone in order to install any Ceph version(>= octopus)
The existence of an "image" option seems to point that cephadm script can be used to deal with different ceph images, and therefore ceph versions.
So "to link" the cephadm script version and the ceph cluster version seems to be incoherent with the existence of this parameter.

Reply:
cephadm is part of core Ceph, not just a software in the vicinity, like ceph-ansible. cephadm is distributed as part of the ceph/ceph container image. cephadm is part of the ceph.spec file. cephadm is part of the RPM repositories alongside with the rest of the ceph packages. 

There is just no way we can use a different version than the rest of ceph. It's the nature of cephadm to be distributed as part of ceph released. 

Imagine ceph-mon would use a different version number. That's just not going to work. No. Let's use the ceph version number.

Reply:
Ok. This clarifies things :-)
Then it seems that distributing cephadm alone (downloading it from url or using an rpm) is a bad practice. We should only 
use the cephadm script included in the ceph image we are using.